Code/src/main/resources/catalog/haskell.xml

```diff
@@ -192,7 +192,7 @@
 
 
   <functions>
-    <category name="Comparison and logic">
+    <category name="Conditionals">
```

Comment:
This catagory contains exactly comparison operators and boolean logic. Conditionals seems unnecessarily vague unless you put guards or other things in there.